frontend/src/app/word-page/word-page.component.html

```
@@ -25,6 +25,21 @@ <h2>Definition</h2>
           <p>No definition</p>
         }
       </div>
+
+      @if (wordEntry.pronunciations?.length) {
+        <mat-divider></mat-divider>
+        <div class="word-section pronunciations">
+          <h2>Aussprache</h2>
+          @for (pronunciation of wordEntry.pronunciations; track pronunciation.url) {
+            <div class="pronunciation">
+              @if (pronunciation.label && wordEntry.pronunciations.length > 1) {
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)